src/guillo-gmi/components/fields/downloadField.js

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react'
+import { useTraversal } from '../../contexts'
+
+export const DownloadField = ({ value }) => {
+  const Ctx = useTraversal()
+  const { data, field } = value
+
+  const getField = async (downloadFile) => {
+    const endpoint = `${Ctx.path}@download/${field}`
+    const res = await Ctx.client.download(endpoint)
+    const text = await res.blob()
+    const blob = new Blob([text], {
+      type: data.content_type,
+    })
+    const url = window.URL.createObjectURL(blob)
+
+    // Create blob link to download
+    const link = document.createElement('a')
+    link.href = url
+    if (downloadFile) {
+      link.setAttribute('download', `${data.filename}`)
+    } else {
+      link.setAttribute('target', `_blank`)
+    }
+
+    document.body.appendChild(link)
+    link.click()
```

Comment:
I think this could generate memory leaks if a [`URL.revokeObjectURL()`](https://developer.mozilla.org/en-US/docs/Web/API/URL/revokeObjectURL) is not called after using the`url`.

```suggestion
    link.click()
    window.URL.revokeObjectURL(url)
```

Reply:
Added! Thanks!